### ISSUE TITLE
Issues concerning users in document sharing

### ISSUE THREAD

**Issue by cyberbaloo** [numerique-gouv/meet#1148]
## Bug Report

**Problematic behavior**
I've grouped the errors in this section to make them easier to fix.

- The SVG present before the user's name is a decorative element.
- Incorrect rendering of the user's role in the document.
- Depending on the document ownership (reader, editor, owner), the user display should not be clickable when one is a reader of the document in particular.

**Expected behavior/code**
- The SVG displaying the user's initials must not be announced by screen readers.
- The user's role is not announced as such. The `aria-label="Document role text"` attribute is announced instead of "Owner" or "Editor".
- Users who have access to the document should be formatted as a list.

**Possible Solution**
- See how it is possible to announce the user's role, taking into account whether one is the document owner, editor, or reader. The display of roles differs depending on document ownership.
`<span aria-label="Texte du rôle du document">Propriétaire</span>`
- Add an `svg aria-hidden="true"` attribute to the initials SVG.
`<svg aria-hidden="true" ...>`
- Two possibilities: either structure with a native HTML list `<ul><li>` or override the semantics with the ARIA technique.

On the listbox container

```
<!-- Before -->
<div role="listbox" aria-label="Suggestions" ...>

<!-- After -->
<div role="list" aria-label="Members with access to the document" ...>
```

On each option item

```
<!-- Before -->
<div role="option" aria-selected="false" aria-disabled="false" ...>

<!-- After -->
<div role="listitem" ...>
```

Remove the `aria-selected` and `aria-disabled` attributes; they don't make sense on a `listitem`

**Additional context/Screenshots**

<img width="790" height="236" alt="Image" src="https://github.com/user-attachments/assets/67c74d86-e6f2-4ebc-9d5c-5b2fdfae96a8" />

**Critères RGAA**

**Critère 1.2 :** [Chaque image de décoration est-elle correctement ignorée par les technologies d’assistance ?](https://accessibilite.numerique.gouv.fr/methode/criteres-et-tests/#1.2)

**Critère 7.1 :** [Chaque script est-il, si nécessaire, compatible avec les technologies d’assistance ?](https://accessibilite.numerique.gouv.fr/methode/criteres-et-tests/#7.1)

**Impact**
The impact is moderate for the decorative image, as the user hears redundant information that doesn't really provide context (how would one know that V D are initials?), and the comprehension of the information may be disrupted, especially if there are multiple users.
The impact is major for the rendering of the owner's role, as the information is not the same when read or announced by the screen reader.

The impact for the last point is major because the information is displayed but the user wastes time believing they cannot select the user.

**Comment by lebaudantoine** [comment]
@cyberbaloo @Ovgodd I think this issue is related to docs
